.rr.yaml

```diff
@@ -0,0 +1,214 @@
+  static:
+    # Path to the directory with static assets.
+    #
+    # This option is required.
+    dir: "./public"
+
+    # File extensions to forbid.
+    #
+    # Default: []
+    forbid: [".htaccess", ".php"]
+
+    # Automatically add headers to every response.
+    #
+    # Default: <empty map>
+    response:
+      X-Powered-By: "RoadRunner"
+
+  # Workers pool settings.
+  pool:
+    # How many worker processes will be started. Zero (or nothing) means the number of logical CPUs.
+    #
+    # Default: 0
+    num_workers: 0
+
+    # Maximal count of worker executions. Zero (or nothing) means no limit.
+    #
+    # Default: 0
+    max_jobs: 64
+
+    # Timeout for worker allocation. Zero means no limit.
+    #
+    # Default: 60s
+    allocate_timeout: 10s
+
+    # Timeout for worker destroying before process killing. Zero means no limit.
+    #
+    # Default: 60s
+    destroy_timeout: 10s
+
+    # Supervisor is used to control http workers (previous name was "limit", docs:
+    # https://roadrunner.dev/docs/php-limit). "Soft" limits will not interrupt current request processing. "Hard"
+    # limit on the contrary - interrupts the execution of the request.
+    supervisor:
+      # Maximal worker memory usage in megabytes (soft limit). Zero means no limit.
+      #
+      # Default: 0
+      max_worker_memory: 128
+
+      # Maximal job lifetime (hard limit). Zero means no limit.
+      #
+      # Default: 0s
+      exec_ttl: 60s
+
+  # HTTP/2 settings.
+  http2:
+    # HTTP/2 over non-encrypted TCP connection using H2C.
+    #
+    # Default: false
+    h2c: false
+
+    # Maximal concurrent streams count.
+    #
+    # Default: 128
+    max_concurrent_streams: 128
+
+## Application metrics in Prometheus format (docs: https://roadrunner.dev/docs/beep-beep-metrics). Drop this section
+## for this feature disabling.
+#metrics:
+#  # Prometheus client address (path /metrics added automatically).
+#  #
+#  # Default: "127.0.0.1:2112"
+#  address: 127.0.0.1:8081
+
+# Health check endpoint (docs: https://roadrunner.dev/docs/beep-beep-health). If response code is 200 - it means at
+# least one worker ready to serve requests. 500 - there are no workers ready to service requests.
+# Drop this section for this feature disabling.
+status:
+  # Host and port to listen on (eg.: `127.0.0.1:2114`). Use the following URL: http://127.0.0.1:2114/health?plugin=http
+  # Multiple plugins must be separated using "&" - http://127.0.0.1:2114/health?plugin=http&plugin=rpc where "http" and
+  # "rpc" are active (connected) plugins.
+  #
+  # This option is required.
+  address: 127.0.0.1:2114
+
+  # Response status code if a requested plugin not ready to handle requests
+  # Valid for both /health and /ready endpoints
+  #
+  # Default: 503
+  unavailable_status_code: 503
+
+service:
+  laravel_queue_worker_1:
+    command: "php /app/artisan queue:work --queue=high,low"
```

Comment:
This has been changed to the following by laravel:
`php artisan queue:work --queue=high,default`

Ref: https://laravel.com/docs/9.x/queues#connections-vs-queues